README.md

```diff
@@ -20,6 +20,19 @@ the search. A results window will pop up with the following keymaps:
 - `<enter>` - accept the result under the cursor
 - `q` - quit out of the quicklink adventure
 
+Additionally, the plugin lets you open a markdown reference (`[reference][]`) by
+pressing `gx`. One may want to change this mapping to anything (`<cr>` for e.g.)
+by modifying the `g:quicklink_open_mapping` global variable like so :
+
+```vim
+g:quicklink_open_mapping = '<cr>'
+```
+
+This functionnality uses `pi_netrw` (`:normal gx`) in order to
+open the link, so it will open it using the program specified by the global
+variable `g:netrw_browsex_viewer` (see `:h netrw-gx`). Also, `:normal gx` will
+behave normally if on normal link.
```

Comment:
Since we have two distinct features here, can we break down the usage section to match. Something like:

``` md
### URL Searching

The main feature of `vim-quicklink` is to search for and insert a relevant
link without needing to leave vim. You can do this by...

...

### Link Opening

`vim-quicklink` also updates the `gx` mapping from `netrw` to work with markdown link format. The default
behavior of `gx` when your cursor is over a URL will still work and is unchanged, but you can now also use
`gx` on a markdown link such as `[webapi-vim][]` in this readme and `vim-quicklink` will find and open the
associated URL using `netrw`.
```